content/blog/2025/custom-functions.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Custom CSS Functions in the Browser
+sub: Start using author-defined functions
+author: miriam
+date: 2025-03-13
+tags:
+  - Article
+  - CSS
+  - Layout
+  - W3C
+image:
+  src: /projects/w3c.jpg
```

Comment:
This seems ok to me. I don't really have any better ideas besides an image of the code or something, which always feels kinda funny.